src/DependencyInjection/Compiler/WellKnownSchemaFilterPass.php

```diff
@@ -21,35 +17,5 @@ class WellKnownSchemaFilterPass implements CompilerPassInterface
     /** @return void */
     public function process(ContainerBuilder $container)
     {
-        $blacklist = [];
-
-        foreach ($container->getDefinitions() as $definition) {
-            if ($definition->isAbstract() || $definition->isSynthetic()) {
-                continue;
-            }
-
-            if ($definition->getClass() !== PdoSessionHandler::class) {
-                continue;
-            }
-
-            $table = $definition->getArguments()[1]['db_table'] ?? 'sessions';
-
-            if (! method_exists($definition->getClass(), 'configureSchema')) {
```

Comment:
since this is alyways true now this class is no-op, right? :eyes:

Reply:
Indeed. Let's remove class too, since you already removed compiler pass definition anyways

Reply:
Yeah although its a BC break I would also tend to remove it as its completely useless now